### PR TITLE
convert deprecated <dl> based WebIDL to Contiguous IDL

### PR DESCRIPTION
**Pull request by patrickhlauke**
see https://www.w3.org/respec/examples/webidl-contiguous.html

this also fixes problem (due to recent respec change?) of `<dd>` without explicit `<p>` not showing up at all

### CODE REVIEW

**Comment by patrickhlauke** [comment]
@rbyers to review ... similar to https://github.com/w3c/pointerevents/pull/101 this resulted in some changes, like no split of "Attributes" and "Methods". dropped the (redundant?) "Dictionary..." bits. most glaring difference is that https://w3c.github.io/touch-events/#methods doesn't show that table with "nullable" etc.

File: index.html [hunk 1]
```diff
@@ -424,68 +427,72 @@ <h3 id="webidl-conform">WebIDL Conformance</h3>
         title="examples">example</a> for sample code demonstrating how to fire
         an untrusted touch event.
       </p>
-
-      <dl title='[Constructor(DOMString type, optional TouchEventInit eventInitDict)] interface TouchEvent : UIEvent' class='idl' data-merge='TouchEventInit'>
-        <dt>readonly attribute <a><code>TouchList</code></a> touches</dt>
+      <pre class="idl">
+dictionary TouchEventInit : EventModifierInit {
+             sequence<Touch> touches = [];
```
Comment (RByers) [inline index.html]: I believe you'll need to use `&gt;` and `&lt;` here for the angle brackets (they're not showing up).  3x

**Comment by RByers** [comment]
Just one little problem, otherwise LGTM.  Thanks!

File: index.html (hunk 1 above)
Comment (patrickhlauke) [inline index.html]: ah, good catch. fixed and force-pushed

**Pull request merged**